pipelines/tasks/StarAlignBamSingleEnd.wdl

```diff
@@ -21,9 +21,12 @@ task StarAlignBamSingleEnd {
       --genomeDir genome_reference \
       --readFilesIn "${bam_input}" \
       --outSAMtype BAM Unsorted \
+      --outFilterMultimapNmax 1 \
```

Comment:
no multiple mapping?

Reply:
Not in the current version of the pipeline. We will address this, but it was removed for the v1 draft.